api/src/main/java/io/opencensus/trace/internal/BaseMessageEventUtil.java

```diff
@@ -14,39 +14,46 @@
  * limitations under the License.
  */
 
-package io.opencensus.internal;
+package io.opencensus.trace.internal;
 
 import static com.google.common.base.Preconditions.checkNotNull;
 
-import io.opencensus.trace.MessageEvent;
+import io.opencensus.common.Internal;
 
 /**
- * Helper class to convert/cast between for {@link MessageEvent} and {@link NetworkEvent}.
+ * Helper class to convert/cast between for {@link io.opencensus.trace.MessageEvent} and {@link
+ * io.opencensus.trace.NetworkEvent}.
  *
  * @since 0.12
  */
+@Internal
 @SuppressWarnings("deprecation")
 public final class BaseMessageEventUtil {
   /**
-   * Cast or convert a {@link io.opencensus.trace.BaseMessageEvent} to {@link MessageEvent}.
+   * Cast or convert a {@link io.opencensus.trace.BaseMessageEvent} to {@link
+   * io.opencensus.trace.MessageEvent}.
    *
    * <p>Warning: if the input is a {@code io.opencensus.trace.NetworkEvent} and contains {@code
    * kernelTimestamp} information, this information will be dropped.
    *
    * @param event the {@code BaseMessageEvent} that is being cast or converted.
    * @return a {@code MessageEvent} representation of the input.
    */
-  public static MessageEvent asMessageEvent(io.opencensus.trace.BaseMessageEvent event) {
+  public static io.opencensus.trace.MessageEvent asMessageEvent(
+      io.opencensus.trace.BaseMessageEvent event) {
     checkNotNull(event);
-    if (event instanceof MessageEvent) {
-      return (MessageEvent) event;
+    if (event instanceof io.opencensus.trace.MessageEvent) {
+      return (io.opencensus.trace.MessageEvent) event;
+    }
+    if (!(event instanceof io.opencensus.trace.NetworkEvent)) { // Work around FindBugs.
+      throw new IllegalArgumentException("NetworkEvent expected.");
```

Comment:
Could you please update findbugs-exclude.xml instead? https://github.com/census-instrumentation/opencensus-java/blob/master/findbugs-exclude.xml#L20

Reply:
Good catch, updated.